components/ReactQueryProvider.tsx

```diff
@@ -0,0 +1,13 @@
+"use client";
+
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import React, { useState } from "react";
+
+const ReactQueryProvider = ({ children }: React.PropsWithChildren) => {
+  const [queryClient] = useState(new QueryClient());
```

Comment:
ASK: 아래와 같은 형태가 아니라 위의 코드처럼 정의하신 이유가 따로 있을까용!?
```suggestion
  const queryClient = new QueryClient();
```

Reply:
루밍이 리뷰에 남겼듯이 Next.js에서는 queryprovider에서 queryclient 정의 시 참조동일성 유지를 위해 useState를 사용한다구 합니당!!